### DISCUSSION TITLE
How do I get 0-day code/template?

### DISCUSSION THREAD

**Discussion by pixelfabs** [adityatelange/hugo-PaperMod#591] [Q&A] [answered]
Don't know how to call it, but how do I download or pull the 0-day code? Like, the code AND the pulls? I don't mind living on the edge and testing the latest features. I had some issues with covers/webp/cls stuff. Most are fixed but not in the code yet, only in pulls?

**Comment by danielfdickinson** [comment, marked as answer]
If you are looking for info on using standard Git versus the stuff GitHub is trying to push on GitHub users, [StackOverflow has info on how to clone a GitHub Pull Request](https://stackoverflow.com/questions/14947789/github-clone-from-pull-request).

If you are using GitHub CLI or GitHub Desktop, it is much easier (I assume) because all you need to do is go to the pull request and select the ``Code`` button and GH CLI or GH Desktop and the instruction will be provided.

If you mean how to you get the combination of the various PR's along with the latest code, I'm afraid that involves you doing the merge, because there does not exist yet such a merge.

If I've missed what you are asking for, please clarify and I'll see if I can help.

Although some people like me have personal forks like this one: https://github.com/danielfdickinson/hugo-PaperMod-dfd-x because we want to use updates without waiting for the maintainer (and are prepared to merge what actually gets accepted by doing the required merge against our fork, once the PRs we are interested in are merged).

**Reply by pixelfabs** [reply]
Damn. merge it is. It's about time I get to learn how to do that. Thanks Daniel :-)
